model/task/db.go

```diff
@@ -616,6 +616,74 @@ func GetRecentTasks(period time.Duration) ([]Task, error) {
 	return tasks, nil
 }
 
+type StatsList struct {
+	Status string   `json:"status"`
```

Comment:
These should have bson tags. They're not strictly necessary, because the driver will implicitly convert type names to lowercase for field names, but in general it's better to be explicit.